certified-connectors/Telstra SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,720 @@
+                  },
+                  "description": "Tag",
+                  "title": "Tags",
+                  "x-ms-visibility": "advanced"
+                }
+              },
+              "required": ["from", "messageContent", "to"]
+            }
+          },
+          {
+            "name": "Telstra-api-version",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "3.x",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Content-Language",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "en-au",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Accept-Charset",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "utf-8",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "x-request-source",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "Telstra SMS Connector",
+            "x-ms-visibility": "internal"
+          }
+        ],
+        "description": "Send SMS/MMS leveraging Australia\u0027s leading mobile network",
+        "x-ms-openai-data": {
+          "openai-enabled": false,
+          "operations": [
+            {
+              "operationId": "SendSMS",
+              "x-ms-require-user-confirmation": true
+            }
+          ]
+        }
+      }
+    },
+    "/virtual-numbers/{virtual-number}/callback": {
+      "x-ms-notification-content": {
+        "schema": {
+          "type": "object",
+          "properties": {
+            "to": {
+              "type": "string",
+              "description": "to"
+            },
+            "from": {
+              "type": "string",
+              "description": "from"
+            },
+            "timestamp": {
+              "type": "string",
+              "description": "timestamp"
+            },
+            "messageId": {
+              "type": "string",
+              "description": "messageId"
+            },
+            "messageContent": {
+              "type": "string",
+              "description": "messageContent"
+            }
+          }
+        },
+        "description": "Response"
+      },
+      "post": {
+        "responses": {
+          "400": {
+            "description": "default",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc. but not default